blackbox/build-and-package.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+TAR_CREATE_OPTS="-exclude-vcs --exclude-vcs-ignores"
```

Comment:
The option `--exclude-vcs` requires two hyphens.

(I just checked mine, and it works; when I run "tar tvf" on the stratisd tarball, I no longer see the '.git' subdirectory.)

Reply:
grr. force-pushed once more.

Reply:
Ok, now it matches what passes my test.